recipes/clarabel/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "clarabel" %}
+{% set version = "0.6.0" %}
+{% set checksum = "ef909a393e72981ca10b1d866d9cc7fb6295ece20ae035def764338894961184" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: 
+      - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+      - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - crossenv                            # [build_platform != target_platform]                   
+    - {{ compiler('rust') }}
```

You don't need `libgcc-ng`, this is the "right" fix for it.

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('rust') }}
```

---

Thank you for the review comments.   And especially for this one, since I had no idea how to resolve that.

---

IMO we should fix this, when a recipe requests `{{ compiler('rust') }}` everything should be in there to make it work. I'm not sure why we still need this. Maybe @isuruf can clarify?

Anyway, until we fix this, adding the c compiler is the right way to go.

---

Do all rust packages additionally need the c compiler, or is it just a function of how pyo3 works in binding rust to the python c-api?